app/config.py

```diff
@@ -317,7 +317,7 @@ class Config(object):
             'send-scheduled-comp-and-pen-sms': {
                 'task': 'send-scheduled-comp-and-pen-sms',
                 # Every 2 minutes past every hour from 13 through 21 on every day-of-month from 22 through end-of-month
-                'schedule': crontab(hour='13-21', day_of_month='22-31', minute='*/2'),
+                'schedule': crontab(hour='13-21', day_of_month='24-31', minute='*/2'),
```

Comment:
thanks for not changing the comment, since we will want to reset this next month.